packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -365,6 +365,60 @@ describe('using cli', () => {
     }
   })
 
+  it('should hide hints with --no-hints', async () => {
+    ctx.fixture('example-project')
+    const data = await ctx.cli('generate', '--no-hints')
+
+    if (typeof data.signal === 'number' && data.signal !== 0) {
+      throw new Error(data.stderr + data.stdout)
+    }
+
+    const engineType = getClientEngineType()
+
+    if (engineType === ClientEngineType.Binary) {
+      expect(data.stdout).toMatchInlineSnapshot(`
+      "Prisma schema loaded from prisma/schema.prisma
+
+      ✔ Generated Prisma Client (v0.0.0, engine=binary) to ./generated/client in XXXms
+      "
+    `)
+    }
+
+    if (engineType === ClientEngineType.Library) {
+      expect(data.stdout).toMatchInlineSnapshot(`
+      "Prisma schema loaded from prisma/schema.prisma
+
+      ✔ Generated Prisma Client (v0.0.0) to ./generated/client in XXXms
+      "
+    `)
+    }
+
+    if (engineType !== ClientEngineType.Library && engineType !== ClientEngineType.Binary) {
```

Comment:
This can never happen, engine type is always either binary or library. Where does this snapshot come from?